bolt/lib/Rewrite/BinaryPassManager.cpp

```diff
@@ -370,7 +375,7 @@ Error BinaryFunctionPassManager::runAllPasses(BinaryContext &BC) {
   else if (opts::Hugify)
     Manager.registerPass(std::make_unique<HugePage>(NeverPrint));
 
-  Manager.registerPass(std::make_unique<ShortenInstructions>(NeverPrint));
+  Manager.registerPass(std::make_unique<ShortenInstructions>(NeverPrint),opts::ShortenInstructions);
```

Comment:
Use clang-format on this line to get correct formatting (it's missing a space after comma). Remember not to format the entire file, just your changes.

Reply:
formatted